test/onnx/verify_onnx.cpp

```diff
@@ -1370,4 +1370,41 @@ TEST_CASE(where_test)
     EXPECT(migraphx::verify_range(result_vector, gold));
 }
 
+std::vector<float> gen_trilu_test(const migraphx::program& p)
+{
+    // input data filled with values 1 to 12
+    std::vector<float> x_data(12);
+    std::iota(x_data.begin(), x_data.end(), 1);
+    migraphx::shape sx{migraphx::shape::float_type, {3, 4}};
+
+    migraphx::parameter_map pp;
+    pp["x"] = migraphx::argument(sx, x_data.data());
+
+    auto result = p.eval(pp).back();
+    std::vector<float> result_vector;
+    result.visit([&](auto output) { result_vector.assign(output.begin(), output.end()); });
+    return result_vector;
+}
+TEST_CASE(trilu_test)
```

Comment:
Can we add more tests ? like number of rows == 1 and with batch size ? 
ONNX doc has extensive list of examples. We can borrow tests from there. https://github.com/onnx/onnx/blob/main/docs/Operators.md#Trilu

Reply:
I added a few other tests, but I will add more at a future date once we get more support for the operator.

Reply:
Current logic would work with batch dimension and also if row ==1. IMO we should add tests in this PR than later, it shouldn't be much work to add those tests.

Reply:
added